fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyMaster.java

```diff
@@ -15,19 +15,27 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import com.ibm.fhir.database.utils.api.AllVersionHistoryService;
 import com.ibm.fhir.database.utils.api.DataAccessException;
 import com.ibm.fhir.database.utils.api.IDatabaseAdapter;
 import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
+import com.ibm.fhir.database.utils.api.ITransactionProvider;
 import com.ibm.fhir.database.utils.api.IVersionHistoryService;
+import com.ibm.fhir.database.utils.common.JdbcConnectionProvider;
+import com.ibm.fhir.database.utils.common.JdbcPropertyAdapter;
 import com.ibm.fhir.database.utils.common.JdbcTarget;
 import com.ibm.fhir.database.utils.common.PrintTarget;
 import com.ibm.fhir.database.utils.model.PhysicalDataModel;
+import com.ibm.fhir.database.utils.pool.PoolConnectionProvider;
+import com.ibm.fhir.database.utils.transaction.SimpleTransactionProvider;
+import com.ibm.fhir.database.utils.version.CreateVersionHistory;
+import com.ibm.fhir.database.utils.version.VersionHistoryService;
 
 /**
  * Set up an instance of Derby for use with unit tests
  */
 public class DerbyMaster implements AutoCloseable {
+    private static final String SCHEMA_NAME = "FHIRDATA";
+    private static final String ADMIN_SCHEMA_NAME = "FHIR_ADMIN";
```

Comment:
Originally I think this class was intended to be generic / agnostic to the specific schema getting created.  This is why its in `fhir-database-utils` and not `fhir-persistence-schema`

Reply:
I totally understand.  any suggestions?

Reply:
Fixed